CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Block selection mode when Control is held while starting a selection
 
+### Changed
+
+- The window is now filled with the `background color` before displaying
```

Comment:
I think I'd move that in the fixed section, since it solves an existing problem.